tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_one_or_more_files_using_pandas.py

```diff
@@ -49,4 +49,7 @@
 print(datasource)
 # </snippet>
 
-assert asset_name in datasource.assets
+try:
+    assert asset_name in datasource.assets
+except:
+    assert asset_name in [asset.name for asset in datasource.assets]
```

Comment:
You might want to get someone more familiar with fluent datasources design to check if this is the canonical way to check for the existence of an asset.

Reply:
The way it's attempting before the change fails because datasource.assets is a list of objects.

Reply:
You can do `datasource.get_asset(get_asset)`. This will raise an error if the asset doesn't exist.

Reply:
I vaguely remember this change and I'm assuming the part under the `try` doesn't work anymore. I think @Kilo59 has the context.

Reply:
This is also not a part of any snippet being shown to the user. I think it's just a sanity check.

Reply:
I like the `datasource.get_asset(asset.name)` approach, could we use that here instead of the try/except?